frontend/cache-handler.js

```diff
@@ -0,0 +1,34 @@
+const { IncrementalCache } = require("@neshca/cache-handler")
+const createRedisHandler =
+  require("@neshca/cache-handler/redis-strings").default
+const createLruHandler = require("@neshca/cache-handler/local-lru").default
+const { createClient } = require("redis")
+
+const client = createClient({
+  url: "redis://localhost:6379", // based on backend/README.md
```

Comment:
```suggestion
url: process.env.REDIS_URL ?? "redis://localhost:6379",
```

This was used by the previous work, so should work fine

Reply:
Done